src/library/librarycontrol.cpp

```diff
@@ -60,114 +62,89 @@ LibraryControl::LibraryControl(Library* pLibrary)
     m_numSamplers.connectValueChanged(SLOT(slotNumSamplersChanged(double)));
     m_numPreviewDecks.connectValueChanged(SLOT(slotNumPreviewDecksChanged(double)));
 
-    // Controls to navigate vertically within currently focussed widget (up/down buttons)
```

Comment:
@jmigual 
It looks like your merge strategy was just "delete everything"...? All of my changes to this file have been deleted.